src/Nethermind/Nethermind.Synchronization/ParallelSync/StaticSelector.cs

```diff
@@ -52,11 +52,13 @@ public StaticSelector(SyncMode syncMode)
             add { }
             remove { }
         }
-        
+
+        public int FastSyncLag => (Current & SyncMode.Beam) == SyncMode.Beam ? SyncModeSelectorConstants.BeamSyncFastSyncLag : SyncModeSelectorConstants.NotBeamSyncFastSyncLag;
```

Comment:
Do we want it to be so dynamic? My first idea was to base it on SyncConfig.BeamSync .

Reply:
> Do we want it to be so dynamic? My first idea was to base it on SyncConfig.BeamSync .

Yeah, but it is StaticSelector used only for tests. We don't have SyncConfig access here.